Gems/AtomLyIntegration/CommonFeatures/Code/Source/Material/EditorMaterialComponentSlot.h

```diff
@@ -20,6 +20,8 @@ namespace AZ
     namespace Render
     {
         //! Details for a single editable material assignment
+        //! When changes are applied to this material slot, it will attempt to forward all of those changes to applicable selected entities
+        //! and their materials
```

Comment:
How does it do this forwarding? I only see it in the new functions you added that take an EntityIdSet. What about functions like ClearProperties()? which only call a bus with m_id? Or is it only supposed to do the forwarding in those special functions? In that case, it should be explained better, to clarify what parts only affect this one slot and what parts might propagate.